vdev/src/commands/release/push.rs

```diff
@@ -0,0 +1,32 @@
+use anyhow::Result;
+use clap::Args;
+
+use crate::app;
+use crate::git;
+use itertools::Itertools;
+
+/// Pushes new versions produced by `make release` to the repository
+#[derive(Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        let version = app::version()?;
```

Comment:
I don't think this is quite right yet. The original script called `cargo vdev version`. That code does a check on the release channel to ensure it is tagged properly. That process should be included here too.

Reply:
I will look into this now